### ISSUE TITLE
Render Most Watched Component as an Ordered List 

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#7707]
**Is your feature request related to a problem? Please describe.**
The most watched component will be rendered as an ordered list so that the list order is announced to screenreader users 

**Describe the solution you'd like**
Update ‘Most watched’ to an ‘ol’ from ‘ul’ on the MAP pages and the Most Watched page

Given I use a screen reader
When I view a section with more than one promo 
And I navigate to the first promo
Then it should be announced as a list item 1 within an ordered list

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by rebeccamcginn** [comment]
Opening for A11y and testing

**Comment by LilyL0u** [comment]
Talkback on Google Pixel 3a (android v10) does not announce numbers as it should, only text. However on the device of @DenisHdz it does, so this could be a strange device specific problem.

Will also check on JAWS, NVDA and ask someone to check on an iPhone.

**Comment by LilyL0u** [comment]
JAWS on IE11 also does not read out the number of the list item. It is announced that there is a list, but when you navigate to the promo it does not announce that it is the first item in a list.

'Heading level one [heading]
List of six items
Graphic [alt text]
Heading level 2 link, video, [heading], duration [duration]
Date'

Straight to next item '[graphic]'
Says 'list end' at the end of the list

There is the same behaviour on the Most Watched component as the page.

NVDA is not working properly, might need to update.

**Comment by LilyL0u** [comment]
After updating NVDA I have now tested this with it.
Similarly to JAWS it announces the list and the number of items, but does not say the number of the item in the list at any point.

**Comment by LilyL0u** [comment]
Comparison to screenreader behaviour of ordered list on http://2bitstudio.co.uk/code-tests/article/list-ordered/ordered-list.html
**JAWS on IE11:** 
_on Example 6 - custom ordered list - list style none_
- 'Heading level 2, [heading]'
- 'List of 3 items'
- 'Madrid right left bracket English right bracket'
- 'Lisbon'
- 'Italy'
- 'List end'

_on Example 7 - custom ordered list - list style none and role list_

Exactly the same as above on example 6.


**NVDA on Firefox**
_on Example 6 - custom ordered list - list style none_
- 'out of list, Heading level 6 [heading]'
- 'List with 3 items, Madrid English'
- 'Lisbon'
- 'Italy'
Says out of list on the heading of the next list.

_on Example 7 - custom ordered list - list style none and role list_

Exactly the same as above on example 6.

**Talkback on Chrome:**
_on Example 6 - custom ordered list - list style none_
- '[heading], heading 2, double tap and hold to long press'
- 'Madrid English, double tap and hold to long press'
- 'Lisbon, double tap and hold to long press'
- 'Italy, double tap and hold to long press'

Bonus note for @greenc05, this 'double tap and hold to long press' is what I kept hearing on the US election tables and now I am wondering if it is something new in talkback as I don't remember it on this list before 🤔 

**Summary:**
Number of list items is announced, but not the number of each item. This is the same as the behaviour on the unordered list on the most watched page and component.

Voiceover still to be tested

**Comment by aeroplanejane** [comment]
Testing on IOS Safari 
**Example 6** 
Announces "heading level 2, Example 6 Custom ordered list - list style none" 
Swiped to list items, announced "Madrid, Lisbon, Bergamo" no list mentioned 

**Example 7** 
Announces "heading level 2, Example 7 - custom ordered list - list style none and role list" 
Swipe to items "List start, Madrid, Lisbon, Bergamo, List End" 

**Most Watched Page**
This is the same as the most watched pages. 

@greenc05 this is expected behaviour right?

**Comment by greenc05** [comment]
So @LilyL0u missing example 7 for Talkback above? I trust that is the same as in the doc? Though example 6 is different in the doc? Mac OS is also outstanding... I've just done that for you and added it to the doc 8) So... we should have some form of list semantics in the Most Watched component that match our testing results for example 7. Thanks people.

**Comment by greenc05** [comment]
@LilyL0u thanks, re 'double tap and hold to long press' must of been an update to your phone? Perhaps it's selected a different verbosity setting for Talkback by default? Interesting that @DenisHdz didn't note it reading this out...

**Comment by LilyL0u** [comment]
I just forgot to add in the line that I also tested example 7 and it was the same as example 6 on talkback. In all cases example 6 and 7 behaved the same.

**Comment by aeroplanejane** [comment]
I am making the assumption that this all as expected and closing this issue. 
If we need to make additional changes to improve the screenreader UX for ordered lists we can raise another issue. 
Thanks for this @greenc05 and @LilyL0u - let me know if you need me to capture this anywhere.
